common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -276,6 +276,14 @@ object CometConf {
       .booleanConf
       .createWithDefault(false)
 
+  val COMET_EXPLAIN_ENABLED: ConfigEntry[String] =
+    conf("spark.comet.explain")
+      .doc(
+        "Set this config to VERBOSE to see logging explaining the reason(s) why a query " +
+          "stage cannot be executed natively with Comet.")
+      .stringConf
+      .createWithDefault("VERBOSE")
```

Comment:
I'm still thinking about what other options we may want here, or if this should just be a boolean config